packages/api-sdk/src/medical/models/network.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+export const networkSchema = z.object({
+  name: z.string().optional(),
+  oid: z.string().optional(),
+  address_line: z.string().optional(),
+  city: z.string().optional(),
+  state: z.string().optional(),
+  zip: z.string().optional(),
```

Comment:
@Orta21 Should we pull this out into an address object or similar?

Reply:
I mean yea we could

Reply:
going to not, since you mentioned CW doesn't have all the same fields.

Reply:
Going to not given that I've dropped some of these fields in the first place.